www/blog/skip_alpha.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Skip alpha
+description: Announce the alpha release of Skip
+slug: skip-alpha
```

Comment:
We can add other metadata here, see e.g. https://docusaurus.io/docs/blog#adding-posts